client/my-sites/stats/checklist-banner/index.jsx

```diff
@@ -67,10 +67,16 @@ export class ChecklistBanner extends Component {
 	};
 
 	handleClose = () => {
+		const { siteId } = this.props;
+		const sitesNeverShowBanner = store.get( storeKeyForNeverShow ) || {};
+		sitesNeverShowBanner[ `${ siteId }` ] = true;
+		store.set( storeKeyForNeverShow, sitesNeverShowBanner );
```

Comment:
Why not use Redux in a case like this?

(Typically, we try to avoid using `store` directly, when we have a more established technique for cases like this. I think/hope we're still planning to phase out our `store` dependency altogether eventually.)